### PR TITLE
Add hillock cells of urethra and prostate

### PR DESCRIPTION
**Pull request by lubianat**
Relates to #1134

### CODE REVIEW

**Comment by anitacaron** [comment]
Side comment: to automatically close the issue you linked, GitHub recognises the words "fixes", and "closes". 🙂

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -22773,6 +22775,24 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "MESH:D000067170"^^xsd:string)
...
 SubClassOf(obo:CL_0017004 ObjectIntersectionOf(obo:CL_0002320 ObjectSomeValuesFrom(obo:BFO_0000051 obo:GO_0120327)))
 
+# Class: obo:CL_0017010 (hillock cell of urethra)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30566875"^^xsd:string) Annotation(oboInOwl:hasDbXref "http://orcid.org/0000-0001-7772-9578"^^xsd:string) obo:IAO_0000115 obo:CL_0017010 "A hillock cell that is located in the urethra.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> obo:CL_0017010 "2022-10-13T13:51:35Z"^^xsd:dateTime)
+AnnotationAssertion(oboInOwl:created_by obo:CL_0017010 <http://orcid.org/0000-0003-2473-2313>)
+AnnotationAssertion(rdfs:label obo:CL_0017010 "hillock cell of urethra")
+EquivalentClasses(obo:CL_0017010 ObjectIntersectionOf(obo:CL_4030024 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000057)))
+SubClassOf(obo:CL_0017010 obo:CL_4030024)
```
Comment (bvarner-ebi) [inline src/ontology/cl-edit.owl:22785]: This axiom can be deleted since it will be inferred from the EA.

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -22773,6 +22775,24 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "MESH:D000067170"^^xsd:string)
 AnnotationAssertion(rdfs:label obo:CL_0017004 "telocyte")
 SubClassOf(obo:CL_0017004 ObjectIntersectionOf(obo:CL_0002320 ObjectSomeValuesFrom(obo:BFO_0000051 obo:GO_0120327)))
 
+# Class: obo:CL_0017010 (hillock cell of urethra)
```
Comment (bvarner-ebi) [inline src/ontology/cl-edit.owl:22778]: Consider adding the requested preferred label as a synonym. Consider adding additional synonyms requested in ticket if there is a dbxref to support.

File: src/ontology/cl-edit.owl [hunk 3]
```diff
@@ -22773,6 +22775,24 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "MESH:D000067170"^^xsd:string)
...
+SubClassOf(obo:CL_0017010 obo:CL_4030024)
+
+# Class: obo:CL_0017011 (hillock cell of prostatic urethra)
+
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> obo:CL_0017011 "2022-10-13T13:58:15Z"^^xsd:dateTime)
+AnnotationAssertion(oboInOwl:created_by obo:CL_0017011 <http://orcid.org/0000-0003-2473-2313>)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30566875"^^xsd:string) Annotation(oboInOwl:hasDbXref "http://orcid.org/0000-0001-7772-9578"^^xsd:string) oboInOwl:hasDbXref obo:CL_0017011 "A hillock cell that is located in the prostatic urethra.")
+AnnotationAssertion(rdfs:label obo:CL_0017011 "hillock cell of prostatic urethra")
+EquivalentClasses(obo:CL_0017011 ObjectIntersectionOf(obo:CL_4030024 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001335)))
+SubClassOf(obo:CL_0017011 obo:CL_0017010)
```
Comment (bvarner-ebi) [inline src/ontology/cl-edit.owl:22794]: This axiom can be deleted since it will be inferred from the EA.

**Review by bvarner-ebi — COMMENTED** [review]
see inline comments

File: src/ontology/cl-edit.owl [hunk 4]
```diff
@@ -22773,6 +22775,24 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "MESH:D000067170"^^xsd:string)
...
+# Class: obo:CL_0017010 (hillock cell of urethra)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30566875"^^xsd:string) Annotation(oboInOwl:hasDbXref "http://orcid.org/0000-0001-7772-9578"^^xsd:string) obo:IAO_0000115 obo:CL_0017010 "A hillock cell that is located in the urethra.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> obo:CL_0017010 "2022-10-13T13:51:35Z"^^xsd:dateTime)
+AnnotationAssertion(oboInOwl:created_by obo:CL_0017010 <http://orcid.org/0000-0003-2473-2313>)
+AnnotationAssertion(rdfs:label obo:CL_0017010 "hillock cell of urethra")
+EquivalentClasses(obo:CL_0017010 ObjectIntersectionOf(obo:CL_4030024 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000057)))
+SubClassOf(obo:CL_0017010 obo:CL_4030024)
+
+# Class: obo:CL_0017011 (hillock cell of prostatic urethra)
```
[the remark this answers is not in the document]
Comment (dosumis) [reply src/ontology/cl-edit.owl:22787]: I think important to have this as a class rather than just a synonym.

**Comment by bvarner-ebi** [comment]
@lubianat, if you'd like to try resolving the conflict, there are notes here:
https://obophenotype.github.io/cell-ontology/resolving_merge_conflicts/

If there are pending questions before merging, let me know and I'll try to assist.

**Comment by bvarner-ebi** [comment]
Hi, @lubianat, checking in if assistance is needed to merge this PR.

**Comment by lubianat** [comment]
Thank you, @bvarner-ebi ! No assistance needed so far, I work on  CL on Thursdays, so there probably will be updates by then. Cheers!

**Comment by bvarner-ebi** [comment]
@lubianat, FYI, I am planning a CL release this coming Friday.

**Comment by lubianat** [comment]
Couldn't do it last week, sorry, and this week I'm away. If anyone wants to
do it faster, its ok for me.

Em ter., 25 de out. de 2022 08:07, bvarner-ebi ***@***.***>
escreveu:

**Comment by bvarner-ebi** [comment]
@shawntanzk, would you mind giving this a review by EOD Thursday so the terms can be included in a Friday release?

**Review by bvarner-ebi — APPROVED** [review]

**Pull request merged**